main.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+# main.py
+import sys
+from parser import parse
+from graph import Graph
+
+if len(sys.argv) != 2:
+    raise Exception('Invalid Input, must be given a file which is in the conllu format')
+
+data = parse(open(sys.argv[1], 'r').read())
+print(data)
```

Comment:
This shouldn't print the entire data, it might be very large and not all relevant to cycle detection. However, it should print those lines of the data that have the dependency cycle.